pages/Reviews/index.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { hex2bin } from '@shopgate/pwa-common/helpers/data';
+import View from 'Components/View';
+import List from 'Components/Reviews/components/List';
+import LoadMoreButton from './components/LoadMore';
+import connect from './connector';
+import {
+  REVIEW_ITEMS_PER_PAGE,
+} from './constants';
+/**
+ *
+ */
+class Reviews extends Component {
+  /**
+   * Context types definition.
+   * @type {{i18n: shim}}
+   */
+  static contextTypes = {
+    i18n: PropTypes.func,
+  };
+  /**
+   * PropTypes definition
+   * @return {{fetchReviews: (*|shim)}}
+   */
+  static propTypes = {
+    fetchReviews: PropTypes.func.isRequired,
+    params: PropTypes.shape().isRequired,
+    reviews: PropTypes.arrayOf(PropTypes.shape()),
+  };
+
+  /**
+   * Reviews are empty array as default.
+   * @return {{reviews: Array}}
+   */
+  static defaultProps ={
+    reviews: [],
+  };
+
+  /**
+   * Fetch the first reviews bundle.
+   */
+  componentDidMount() {
+    const { productId } = this.props.params;
+    // ProductId comes from url here, need to encode it first.
+    this.props.fetchReviews(hex2bin(productId), REVIEW_ITEMS_PER_PAGE, 0);
```

Comment:
We want to move away from lifecycle hooks calling for data. Data fetching should now occur in the Observable Streams.